vignettes/custom-expectation.Rmd

```diff
@@ -21,6 +21,35 @@ This vignette shows you how to write your own expectations. Custom expectations
 
 In this vignette, you'll learn about the three-part structure of expectations, how to test your custom expectations, see a few examples, and, if you're writing a lot of expectations, learn how to reduce repeated code.
 
+## Do you need it?
+
+But before you read the rest of the vignette and dive into the full details of creating a fully 100% correct expectation, consider if you can get away with a simpler wrapper. For example, take this expectation from tidytext:
```

Comment:
why not start with an example of a wrapper that is fine? For instance it could be 

```r
expect_df <- function(tbl) {
  expect_s3_class(tbl, "data.frame")
}
```

Or here is a real example: https://github.com/ropensci/aorsf/blob/2b7fa72bff5cd4e8b2984358e35d0938e50540b0/tests/testthat/helper-orsf.R#L259

Reply:
Good idea, thanks!